spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,178 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.camunda.bpm</groupId>
+    <artifactId>camunda-root</artifactId>
+    <version>7.13.0-SNAPSHOT</version>
+    <relativePath>../</relativePath>
+  </parent>
+
+  <groupId>org.camunda.bpm.springboot.project</groupId>
+  <artifactId>camunda-bpm-spring-boot-starter-root</artifactId>
+  <name>camunda BPM - Spring Boot Starter - Root Pom</name>
+  <inceptionYear>2015</inceptionYear>
+
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>starter</module>
+    <module>starter-rest</module>
+    <module>starter-webapp-core</module>
+    <module>starter-webapp</module>
+    <module>starter-webapp-ee</module>
```

Comment:
How will this work for CE users trying to build the starter?

Reply:
It won't, I guess. Question is, if it needs to for now?
In the original starter repo, this was only triggered on release (when an environment variable `RELEASE_VERSION` was set). It would like to have this run in the CI build as well. What do you think?
We will have a follow-up task to transfer the EE part(s) to the platform EE repo in the near future.
@ThorbenLindhauer, should we provide an intermediate solution if we potentially move the EE parts soon anyway?

Reply:
Let's accept it for the time being and then move the EE artifacts to the EE repo.